### PR TITLE
[LiveLogger] Report build time

### LINKED ISSUE #8450: [LiveLogger] logger should report build time.
LiveLogger should report build time after the build, as it is done by Console Logger:
```
Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.89
```

### PR DESCRIPTION
**Pull request by edvilme**
Fixes #8450

### Context
The console logger shows the total time elapsed for the build in the summary. It should also be shown on the summary for the LiveLogger

### Changes Made
The difference between the time at build start and build end is displayed in the summary at the end of the build.

### Testing
Manual testing

### CODE REVIEW

File: src/MSBuild/LiveLogger/LiveLogger.cs [hunk 1]
```diff
@@ -119,10 +122,12 @@ private void UpdateFooter()
         // Build
         private void eventSource_BuildStarted(object sender, BuildStartedEventArgs e)
         {
+            startTime = DateTime.Now;
```
Comment (Forgind) [inline src/MSBuild/LiveLogger/LiveLogger.cs:125]: Technically, I think these should be UTCNow...I'm not kidding when I say I saw a bug at one point from someone who built in different time zones and saw errors as a result. It 100% shouldn't matter in this case, but just thought I'd mention it.

**Review by Forgind — APPROVED** [review]
Looks good!

File: src/MSBuild/LiveLogger/LiveLogger.cs (hunk 1 above)
Comment (rainersigwald) [reply src/MSBuild/LiveLogger/LiveLogger.cs:125]: I'd think about calling and storing [`Stopwatch.StartNew()`](https://learn.microsoft.com/dotnet/api/system.diagnostics.stopwatch.startnew) instead. It's designed to deal with exactly the monotonic-clock situation, though it generally has more accuracy than we care about for build scenarios (we care about seconds, and ticks/15ms timer accuracy are irrelevant).

File: src/MSBuild/LiveLogger/LiveLogger.cs [hunk 2]
```diff
@@ -311,15 +317,15 @@ public void Shutdown()
...
             else
             {
                 Console.WriteLine(ANSIBuilder.Formatting.Color("Build failed.", ANSIBuilder.Formatting.ForegroundColor.Red));
-                Console.WriteLine($"\t{warningCount} Warnings(s)");
-                Console.WriteLine($"\t{errorCount} Errors(s)");
             }
+            Console.WriteLine($"\t{warningCount} Warnings(s)");
+            Console.WriteLine($"\t{errorCount} Errors(s)");
+            Console.WriteLine();
+            Console.WriteLine($"Time elapsed {buildDuration.ToString()}");
```
Comment (rainersigwald) [inline src/MSBuild/LiveLogger/LiveLogger.cs:328]: I definitely want this rounded down, nobody cares about their build time to the ten-thousandth of a second :)

But you got me thinking, how do we feel about something like this instead?

image

Comment (Forgind) [reply src/MSBuild/LiveLogger/LiveLogger.cs:328]: I'm good with that 🙂

Comment (edvilme) [reply src/MSBuild/LiveLogger/LiveLogger.cs:328]: So you mean only minutes:seconds and not fractions of a second?

**Comment by rainersigwald** [comment]
I pushed a couple of changes to riff on/think about.

**Review by Forgind — APPROVED** [review]

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**